groups/models.py

```diff
@@ -161,6 +167,9 @@ def delete_state(self):
     def is_deleted(self):
         return self.state == self.STATE_DELETED
 
+    def __str__(self):
+        return 'Comment by {}'.format(self.user.get_full_name())
```

Comment:
Generally speaking, it's a bad plan to draw information from related models if it can be helped. As an alternative, I'd suggest that this be defined more generically (given it's a polymorphic model)
```
    def __str__(self):
        return '{} on Discussion #{}'.format(self.__class__.__name__, self.discussion_id)
```